packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -145,7 +145,8 @@ class ProjectType(enum.Enum):
         "trashed_at",
         sa.DateTime(timezone=True),
         nullable=True,
-        doc="Timestamp indicating when the project was marked as trashed, or null otherwise.",
+        doc="The date and time when the project was marked as trashed. Null if the project has not been trashed."
+        "When a project is implicitly trashed (e.g. a parent is trashed) then it also has hidden=True",
```

Comment:
so a trashed project gets hidden?
and a project trashed because its parent? was trashed as well? what is this use-case? we are talking about api projects? or projects from folders?

Reply:
> so a trashed project gets hidden?

explicit -> (trashed=True, hidden=False )
implicit -> (trashed=True, hidden=True )

> project trashed because its parent? 

that is what we denote implicit trash

> was trashed as well?  what is this use-case?

yes, it is indeed trash. The difference is whether it shows or not in the trash-bin. Note that the front-end uses the listing operation (`GET /folders?filters={trashed:true}` and analogous with projects) to get the items of the trash bin

> we are talking about api projects? or projects from folders?

both. See the api entrypoits in the highlights of the PR description

Reply:
hmm I don't see anything about public API projects, that might have another project as parent

Reply:
--> new column 'trashed_explicitely'

Reply:
added